package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@teclib/flyve-mdm-android-inventory-agent",
   "version": "2.0.0",
-  "description": "Flyve Inventory Agent for Android",
+  "description": "Flyve MDM Inventory Agent for Android",
```

Comment:
It's not a "refactor" commit type, you can use "ci", because the package JSON is used to generate the version number and changelog.

Reply:
Refactor is used for the Java source code in this case.

Reply:
Refactor is used for the Java source code in this case.